### ISSUE TITLE
Cannot access python library `active_subspaces`

### ISSUE THREAD

**Issue by maemory** [paulcon/as-data-sets#1]
Was trying to run the notebook for the `HyShot II`, however it imports a library `active_subspaces`. It might be nice to include this library in the Github repo somewhere.

**Comment by paulcon** [comment]
Yeah, I just put the notebooks on github so I could use nbviewer. Working
on the code. Stay tuned.
